### ISSUE TITLE
Kvazaar only encodes the first 15 frames

### ISSUE THREAD

**Issue by bayantaani** [ultravideo/kvazaar#183]
Hello,

I am trying to encode a video file that is slightly more than 1 minute long. When I use Kvazaar it only encodes the first 15 frames and then stops normally without any errors.

I tries specifying the number of frames by setting --frames 120 and it did the same thing. How can I fix this?

**Comment by shen-lee** [comment]
Hello,


I thought that you should change your setting '--frames 120' by '-n 120'.
Just have a try. ^_^


On 12/28/2017 13:55，bayantaani<notifications@github.com> wrote：

Hello,

I am trying to encode a video file that is slightly more than 1 minute long. When I Kvazaar it only encodes the first 15 frames and then stops normally without any errors.

I tries specifying the number of frames by setting --frames 120 and it did the same thing. How can I fix this?

**Comment by bayantaani** [comment]
I already tried that, same result :(

I also tried with the option --seek to see if it only encodes the first 15 frames, and when I specify any value larger than 14 (which is frame no. 15) it does not encode anything! it's like the video contains only the first 15 frames.

**Comment by shen-lee** [comment]
Check your input YUV file. As I know, if you don't set frame number, it will always encode before the end of the file. 
On 12/28/2017 13:59，bayantaani<notifications@github.com> wrote：

I already tried that, same result :(

I also tried with the option --seek to see if it only encodes the first 15 frames, and when I specify any value larger than 14 (which is frame no. 15) it does not encode anything! it's like the video contains only the first 15 frames.

**Comment by shen-lee** [comment]
Hello, 
I don't know the version you use. I have tried 0.20,0.24,0.30 and 1.20.
They work well, as follow(1.2.0):
Sorry that YUV file is to large to upload, I suggest you to change a YUV file and then have a try.


On 12/28/2017 14:04，李申<shen_lee@126.com> wrote：
Check your input YUV file. As I know, if you don't set frame number, it will always encode before the end of the file. 
On 12/28/2017 13:59，bayantaani<notifications@github.com> wrote：

I already tried that, same result :(

I also tried with the option --seek to see if it only encodes the first 15 frames, and when I specify any value larger than 14 (which is frame no. 15) it does not encode anything! it's like the video contains only the first 15 frames.
